### DISCUSSION TITLE
Dashboard Canarytrace 🥃 RUM

### DISCUSSION THREAD

**Discussion by lucienevludova-direct** [canarytrace/documentation#212] [RUM] [answered]
Ahoj, 
chtěla jsem se zeptat, máme dnes přidaný a vložený dashboard Canarytrace 🥃 RUM. Některé vizualizace v dashboardu nám nefungují, většinu se nám podařilo upravit, ale nikde v c.rum.metrics-* nemohu dohledat pole view.actions.*. Nevím, zda špatně hledám, nebo máme něco špatně nastaveno. Je možné, že nám zatím nechodí žádná data akcí, ale z ostatních polí bych čekala, že tam bude, ale bude prázdné, respektive bez hodnot.
Děkuju, 
Lucka

**Comment by rdpanek** [comment, marked as answer]
@lucienevludova-direct Ahoj, ano, pokud nejake pole v payloadu chybi, tak vubec nechodi, takze zadna action nebyla spustena.
Vizualizace nefungují, pokud nemají dostupná konkrétní labels, podle kterých třeba agregují.
